wiki/People/The_Team/Nomination_Assessment_Team/SEV_rating/en.md

```diff
@@ -0,0 +1,167 @@
+# SEV rating
+
+SEV is a measurement system used by the [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) (*NAT*) to rate how relevant a [nomination reset](/wiki/Beatmap_ranking_procedure#nomination-resets) is to the evaluations of the nominating [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) (*BN*), providing both transparency and consistency. This measurement is split up into two values and displayed as "*Obviousness/Severity*". Both values range from 0 to 2, making it a simple to understand and quick to use system.
+
+***Important to note:*** *A reset done to fix things which would be considered fine if left unfixed are always rated 0/0. This is to not discourage modding and applying improvements in [Qualified](/wiki/Beatmap/Category#qualified).*
+
+## Disclaimers
+
+1. This article is **NOT a guide**, it's a historical archive that's meant to aid understanding of SEV.
+2. All of the cases mentioned in [Common reset reasons](#common-reset-reasons) are **NOT examples**, they are real resets from the osu! game mode.
+3. The purpose of this article is **NOT to be strictly followed by the NAT**, it's to give a point of reference for how the NAT has historically and statistically handled SEV ratings.
```

Comment:
After thinking about this for a bit, let's remove this entirely. I think this being copied onto the page was a bit of a misunderstanding, as it was more directed internally.

1. & 3. The article could actually be treated as a guide, pretty sure I more meant to say that it shouldn't be blindly followed. We stated that we handle everything case-by-case and that ratings may differ depending on context, so there should be no reason for people to think we follow this strictly, unless they just didn't read.
    - (Which someone didn't in the past, leading to a big discussion that immediately ended when I pointed out that part of the text - honestly don't think this is the document's fault; we shouldn't need to emphasize it more.)
3. We already stated that they are real cases right before we list them. This point was more directed at us previously having referred to the cases as "examples", and wasn't meant to make it onto the page.

```suggestion
```